pkgs/top-level/all-packages.nix

```diff
@@ -433,6 +433,10 @@ with pkgs;
 
   checkpwn = callPackage ../tools/misc/checkpwn { };
 
+  vera = callPackage ../development/tools/analysis/vera { };
+
+  vera-epitech = callPackage ../development/tools/analysis/vera/epitech.nix { };
```

Comment:
I think there should be another opinion on this, but maybe consider renaming this to `banana-vera` ?

Epitech is the upstream maintainer, but the fork itself is called `banana-vera`